File: JavaScript/5kyu/RGBToHexConversion.js
function rgb(r, g, b) {
  return r.hex() + g.hex() + b.hex();
}

Number.prototype.hex = function () {
  var value = Math.max(0, Math.min(255, this));
  return value.toString(16);
};

console.log(rgb(173, 255, 47));

File: CSharp/5kyu/RGBToHexConversion.cs
using System.Linq;
using System;

namespace CSharp._5kyu
{
    public static partial class Kata5
    {
        public static string Rgb(int r, int g, int b)
        {
            return String.Format("{0:X2}{1:X2}{2:X2}", r.Rnd(), g.Rnd(), b.Rnd());
        }

        private static int Rnd(this int rgb)
        {
            return Math.Min(255, Math.Max(0, rgb));
        }
    }
}